ocaml/xapi/xapi_mgmt_iface.ml

```diff
@@ -117,6 +118,49 @@ end = struct
     mode := next_mode
 
   let current_mode () = !mode
+
+  let is_ipv6_enabled () = !ipv6_enabled
+end
+
+module Client_certificate_auth_server = struct
+  let management_server = ref None
+
+  let must_be_running ~__context ~mgmt_enabled =
+    let pool = Helpers.get_pool ~__context in
+    mgmt_enabled
+    && Pool_role.is_master ()
+    && Db.Pool.get_client_certificate_auth_enabled ~__context ~self:pool
+
+  let configure_port cmd =
+    let cmd' = if cmd then "open" else "close" in
```

Comment:
I would prefer a `match` to translate a `cmd` into a string but obviously this works as along as `cmd` is a boolean.

Reply:
I do prefer an old-fashioned if statement sometimes :) Especially for simple Boolean checks. A matter of taste I guess.